suspicious_subjects.txt

```diff
@@ -21,43 +22,61 @@ document has been sent to you via docusign
 document is ready for signature
 docusign
 encrypted message
+encrypted secured
+efax
+expir
 failed delivery
 fedex tracking
 file was shared
 freefax
 fwd: due invoice paid
 has shared
+important information
 inbox is full
+infected
 invitation to comment
 invitation to edit
 invoice due
 left you a message
+low storage
 message from
 message received
 new evoice
+new document
+new fax
+new file
 new message
+new sign-in
+new sign in
 new voicemail
 on desk
 out of space
+password maintenance
 password reset
+payment receipt
 payment status
+payroll disbursement
 pending invoice
+processed
```

Comment:
how confident are you on the list? This definitely doesn't seem standalone in a rule given phrases like this. And tips on how the list should be used that we can capture in the description?

Reply:
this list definitely shouldn't be used standalone, it should be combined with other signals. here's a sample usage: https://github.com/sublime-security/sublime-rules/blob/6c7108f3d644fe68efb07697cddc2462eafece81/detection-rules/link_credential_phishing_intent_and_other_indicators.yml

will update the description!

Reply:
https://github.com/sublime-security/static-files/pull/56/commits/a17dd48d4e30b124bfa0d746edde3f5e2861251c

Reply:
sweet thanks! this also seems like an excellent list to bootstrap ML. I could see a future version of NLU where it works in small subject lines too